### PR TITLE
Submitting Mixer

### PR DESCRIPTION
**Pull request by ChildishGiant**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. 
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. N/A

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.github.childishgiant.mixer

**Comment by hfiguiere** [comment]
That's what I thought: there is no `io.elementary.Sdk` 

You probably instead to use the BaseApp that's off org.freedesktop.Sdk. This is what other Elementary apps do for Flathub.

**Comment by hfiguiere** [comment]
bot, build com.github.childishgiant.mixer

**Comment by ChildishGiant** [comment]
bot, build com.github.childishgiant.mixer

**Comment by ChildishGiant** [comment]
Hmm it still can't find libhandy :thinking:

**Comment by hfiguiere** [comment]
> Hmm it still can't find libhandy 🤔

that's because it's not build before. modules are build in order, so the app should be last.

**Comment by ChildishGiant** [comment]
bot, build com.github.childishgiant.mixer

File: com.github.childishgiant.mixer.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - name: libhandy
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/libhandy.git
+        tag: 1.2.3
+        commit: e9ca4ac4b134bbd6ab91b6f916a915192fd3a465
+    config-opts:
+      - -Dglade_catalog=disabled
+      - -Dvapi=false
```
Comment (hfiguiere) [inline com.github.childishgiant.mixer.yml:41]: it's a Vala you are building.

```suggestion
      - -Dvapi=true
```

(just a guess)

**Comment by ChildishGiant** [comment]
bot, build com.github.childishgiant.mixer

**Comment by ChildishGiant** [comment]
Thanks for all the help on this, there's no way I would have worked all this out myself

**Pull request closed without merging**